growattServer/__init__.py

```diff
@@ -54,8 +56,10 @@ def __get_date_string(self, timespan=None, date=None):
         date_str=""
         if timespan == Timespan.month:
             date_str = date.strftime('%Y-%m')
-        else:
+        elif timespan == Timespan.day or timespan == Timespan.hour:
             date_str = date.strftime('%Y-%m-%d')
+        else:
```

Comment:
Could you reorder the if statement to have %Y at the top, %Y-%m after the and %Y-%m-%d at the bottom? Helps with reading through the code faster 🙂